File: Go/src/Problem001.go
package euler

import (
	"fmt"
	"time"
)

func Problem001() {

	start := time.Now()
	ans := 0
	for i := 2; i < 1000; i++ {
		if i%5 == 0 || i%3 == 0 {
			ans += i
		}
	}
	end := time.Now()
	fmt.Printf("\nAnswer to Problem 1 : %d\n", ans)
	fmt.Printf("Time Taken: %f seconds\n\n", end.Sub(start).Seconds())

}

File: Rust/solutions/src/problem001.rs
use std::time::{Instant};


pub fn problem001() {
    let mut sum = 0;
    let start = Instant::now();

    for i in 1..1000 { if i%3 == 0 || i%5 == 0 { sum += i; } }

    println!("Answer: {}", sum);
    println!("Time Taken: {}", start.elapsed().as_millis());
}
